Documentation~/Blender.md

```diff
@@ -94,3 +95,32 @@ The following is a list of supported nodes and how they are handled.
 | Normal Map | Normal strength is exported. The normal map needs to be in tangent space. |
 | Image Texture | Images are exported but not the texture coordinates. Unity uses UV0. |
 | Displacement | Height and scale is exported. |
+
+
+## Material baking
+
+The basic material sync mode only supports bsdf inputs that are textures or constant values.
+MeshSync has functionality to create baked material copies for each object to allow syncing of procedural materials.
+
+|**Option** |**Description** |
+|:---     |:---|
+| Objects to bake | Bakes all objects in the scene or only selected objects (including hidden objects). |
+| Material channels to bake | Choose which channels should be baked. |
+| Bake to individual materials | Performs the bake. |
+| Restore original materials | Removes the baked material copies and assigns the original materials back to all objects in the scene. |
+| Baked texture path | Folder to save baked textures in. |
+| Baked texture size | Baked texture dimensions. |
+
+MeshSync will attempt to find a BSDF node connected to the *Material Output* node and bake the input of the BSDF. If there is no BSDF connected, MeshSync will bake the data coming into the *Material Output* that can be baked (Only color, normals and roughness are supported). 
+Shader nodes that take other shader nodes as input (Mix and Add Shader) cannot be baked and will use the fallback mode.
+
+If the object has no UVs, MeshSync will use blender's *Smart UV Project* operator to generate UVs.
+
+The blender console will show progress during the bake.
+
+## Material baking troubleshooting
+|**Problem** |**Possible cause** |
+|:---     |:---|
+| The baked maps are black | There are many possible causes. Is the object UV unwrapped? Baking will not work without valid UVs. Metallic materials do not bake well in blender. |
+| The baked maps do not match the original material | The material node tree does not use a BSDF as input to the *Material Output* node and the lower quality fallback bake mode had to be used. |
+| Geometry node instances are missing | To bake materials, all modifiers need to be applied. When geometry node modifiers are applied, instances are lost. Use a *Realize Instances* node to make the instances real geometry. |
```

Comment:
Does the baking script apply all the modifiers? is that reversed after the baking?

Reply:
can we not just bake on the evaluated objects?

Reply:
No, the bake is a blender operation that works on the scene objects.